example/src/components/ChatboxTest.tsx

```diff
@@ -18,118 +18,127 @@ import {
   GRAASP_PANEL_WIDTH,
 } from '../config/constants';
 
-const useStyles = makeStyles((theme) => ({
-  testContainer: {
-    padding: theme.spacing(2),
-    background: 'lightblue',
-  },
-}));
-
 type Props = {};
 
 const ChatboxTest: FC<Props> = () => {
-  const classes = useStyles();
-  const [showTools, setShowTools] = useState(false);
   const [testWidth, setTestWidth] = useState(GRAASP_PANEL_WIDTH);
-  const [showHeader, setShowHeader] = useState(false);
+  const [showTools, setShowTools] = useState(false);
   const [lang, setLang] = useState(DEFAULT_LANG);
   const [chatId, setChatId] = useState(DEFAULT_CHAT_ID);
 
+  const useStyles = makeStyles((theme) => ({
+    container: {
+      display: 'flex',
+      flexDirection: 'row',
+    },
+    chatboxContainer: {
+      margin: theme.spacing(1),
+      width: testWidth,
+      padding: theme.spacing(0, 1),
+      border: 'solid darkblue 1px',
+      borderRadius: '6px',
+    },
+    testContainer: {
+      margin: theme.spacing(1),
+      display: 'flex',
+      flexDirection: 'column',
+      borderRadius: '6px',
+      padding: theme.spacing(2),
+      width: '400px',
+      border: 'solid darkred 1px',
+      '& > *': {
+        margin: theme.spacing(2, 0) + '!important',
+      },
+    },
+    textInputControl: {
+      width: '100%',
+    },
+    chatInputBox: {
+      backgroundColor: 'white',
+      // borderRadius: '8px',
+      // border: 'solid red 2px',
+    },
+  }));
+
+  const classes = useStyles();
+
   // adapt the width of the chatbox to simulate the width used on Graasp
-  const onCheckPanelWidth = (e: React.ChangeEvent<HTMLInputElement>) => {
-    if (Number(e.target.value) === 0) {
-      setTestWidth(GRAASP_PANEL_WIDTH);
+  const onChangePanelWidth = (_: unknown, newValue: number | number[]) => {
+    if (typeof newValue == 'number') {
```

Comment:
`Number.isInteger()`? or similar

Reply:
I was just trying to narrow the type to `number` so that TS would not complain. In the current setup i will always receive only `number` types for the slider but i had to put the function argument like this to that it would not complain... What do you think @pyphilia ?